include/rest/api_client.hpp

```diff
@@ -144,19 +243,11 @@ namespace disccord
                     }
 
                     template <typename TModel>
-                    pplx::task<void> request_multi_json(route_info& route, std::vector<TModel> body, const pplx::cancellation_token& token = pplx::cancellation_token::none())
+                    pplx::task<void> request_multi_json(route_info& route, TModel body, const pplx::cancellation_token& token = pplx::cancellation_token::none())
```

Comment:
there is no endpoint that needs this request though. as in: the user building a single model and wanting to send up an object, why turn it into an array? there are only 2 endpoints in the API where we need to send up a json array as the body. the rest will that would use `request_multi_json` would only send up an object, and can expect empty, object, or an array as a response. the reason I made a new request type is because we need to suppose the above, but also need a request where the incoming body is already a list (since the user will be appending to it at a higher level before they make the request). hence why I have `request_json_array` as well. maybe the naming convention is weird which lead to my confusion, but regardless, we still do need to support these two types of requests I mentioned.